components/list/list-item-mixin.js

```diff
@@ -530,6 +586,17 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 		this._hovering = false;
 	}
 
+	_onNestedSlotChange() {
+		if (this.selectable) {
+			this._onNestedSlotChangeCheckboxMixin();
+		}
+		const nestedList = this._getNestedList();
+		if (this._hasNestedList !== !!nestedList) {
+			this._hasNestedList = !!nestedList;
+			this.dispatchEvent(new CustomEvent('d2l-list-item-nested-change', { bubbles: true, composed: true }));
```

Comment:
I this this is consider an internal/private event, ya? If so, this should have `/** @ignore */` on the line preceding the `dispatch` call, [such as here](https://github.com/BrightspaceUI/core/blob/main/mixins/labelled-mixin.js#L184). It wouldn't have a description, but it would still show up in the generated Daylight docs.